skillserver/skillserver.go

```diff
@@ -136,10 +145,9 @@ func RunSSL(apps map[string]interface{}, port, cert, key string) {
 	log.Fatal(srv.ListenAndServeTLS(cert, key))
 }
 
-func initialize(apps map[string]interface{}, router *mux.Router) {
-	flag.BoolVar(&insecureSkipVerify, "insecure-skip-verify", false, "Skip certificate checks for downloading from AWS")
-	flag.Parse()
-	if insecureSkipVerify {
+func initialize(apps map[string]interface{}, router *mux.Router, isv bool) {
+	insecureSkipVerify = isv
+	if isv {
```

Comment:
Why not just name the variable in the function declaration "insecureSkipVerify"?

Reply:
because insecureVerify is a package variable which needs to be set. If the argument is called `insecureSkipVerify` it shadows the package variable. Did this first by mistake ;-) 

But I try to make it cleaner.